x-pack/filebeat/module/checkpoint/firewall/ingest/pipeline.json

```diff
@@ -307,56 +307,56 @@
     {
       "append": {
         "field": "event.category",
-        "value": "malware",
+        "value": ["malware"],
```

Comment:
Not wrong, but it is more cycles & memory to append arrays than it is to add a single element to an array.  So you could just to leave the value as "malware" instead of ["malware"]

Reply:
Modified the different fields for this